test/common/http/http2/http2_connection.proto

```diff
@@ -0,0 +1,95 @@
+syntax = "proto3";
+
+package test.common.http.http2;
+import "test/fuzz/common.proto";
+
+message Http2DataFrame {
+  bytes data = 1;
+  // Use only 255 bytes maximum
+  bytes padding = 2;
+}
+
+message Http2HeaderFrame {
+  uint32 stream_dependency = 1;
+  uint32 weigth = 2;
+  test.fuzz.Headers headers = 3;
+  bytes padding = 4;
+}
+
+message Http2PriorityFrame {
+  uint32 stream_dependency = 1;
+  uint32 weight = 2;
+}
+
+message Http2RstFrame {
+  uint32 error_code = 1;
+}
+
+message Http2Setting {
+  // Uses only lower 2 bytes
+  uint32 identifier = 1;
+  uint32 value = 2;
+}
+
+message Http2SettingsFrame {
+  repeated Http2Setting settings = 1;
+}
+
+message Http2PushPromiseFrame {
+  uint32 stream_id = 1;
+  test.fuzz.Headers headers = 2;
+  bytes padding = 3;
+}
+
+message Http2PingFrame {
+  uint64 data = 1;
+}
+
+message Http2GoAwayFrame {
+  uint32 last_stream_id = 1;
+  uint32 error_code = 2;
+  bytes debug_data = 3;
+}
+
+message Http2WindowUpdateFrame {
+  uint32 window_size_increment = 1;
+}
+
+message Http2ContinuationFrame {
+  test.fuzz.Headers headers = 1;
```

Comment:
Not sure how I feel about this, as the payload of a CONTINUATION frame is likely to be a subrange of a deflated header block, rather than a complete header block unto itself. I understand that we'd like to fuzz this frame type, so I don't have a better suggestion.